var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -422,6 +422,8 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         depends_on("kokkos-kernels@4.3.01", when="@16")
         depends_on("kokkos-kernels@4.2.01", when="@15.1:15")
         depends_on("kokkos-kernels@4.1.00", when="@15.0")
+        depends_on("kokkos+openmp", when="+openmp")
+        depends_on("kokkos-kernels+openmp", when="+openmp")
```

Comment:
In general I don't think you need the second line here, Kokkos Kernels is not really turning on or off openmp, we just use what Kokkos is built with

Reply:
Hi Luc, thanks for the note! Let me just double check that removing this second line still makes everything work as expected

Reply:
At least with `@15.0.0` I can confirm I did not need ` depends_on("kokkos-kernels+openmp", when="+openmp")`

Reply:
Thank you, Chris! I can confirm that from my end as well